src/App.css

```diff
@@ -1,3 +1,12 @@
 .App {
   text-align: center;
 }
+
+.home,
```

Comment:
I see you are setting the css for each of the subcomponents of the `App` component in the `App.css` file. 
I'd move css related to each component to its own file.